### ISSUE TITLE
Updated manuscript: add D3b/DBHI authors to date

### ISSUE THREAD

**Issue by jharenza** [AlexsLemonade/OpenPBTA-manuscript#107]
#### What section of the manuscript should be updated?
Author list

#### Proposed changes
This issue is being opened to track D3b authors and roles for the OpenPBTA manuscript.

#### Relevant literature
We will use [CRediT taxonomy](https://casrai.org/credit/). Alternate [link](https://www.cell.com/pb/assets/raw/shared/guidelines/CRediT-taxonomy.pdf).

Below, we will add full author names for the roles below and then file a PR:

| Role                       | Authors |
|----------------------------|---------|
| Conceptualization          |    Jo Lynne Rokita, Pichai Raman, Adam C. Resnick, Jena V. Lily, Philip B. Storm    |
| Data curation              |    Jo Lynne Rokita, Krutika S. Gaonkar, Yuankun Zhu, Miguel A. Brown, Brian M. Ennis, Bo Zhang, Bailey Farrow, Nick VanKuren, Payal Jain  |
| Formal Analysis            |   Jo Lynne Rokita, Krutika S. Gaonkar, Yuankun Zhu, Tejaswi Koganti, Shrivats Kannan, Nhat Duong, Komal S. Rathi, Pichai Raman, Daniel P. Miller, Mateusz P. Koptyra, Miguel A. Brown, Bo Zhang, Brian M. Ennis     |
| Funding acquisition        |   Adam C. Resnick, Philip B. Storm, Jo Lynne Rokita, Jena V. Lilly      |
| Investigation              |    Jo Lynne Rokita, Krutika S. Gaonkar, Nhat Duong, Komal S. Rathi, Yuankun Zhu, Tejaswi Koganti, Mariarita Santi, Angela Viaene, Miguel A. Brown, Payal Jain    |
| Methodology                |     Jo Lynne Rokita, Krutika S. Gaonkar,  Shrivats Kannan, Nhat Duong, Hongbo M. Xie, Komal S. Rathi, Pichai Raman, Yuankun Zhu, Miguel A. Brown, Adam A. Kraya   |
| Project administration     |   Jena V. Lilly, Allison P. Heath   |
| Resources                  |    Adam C. Resnick, Philip B. Storm    |
| Software                   |    Jo Lynne Rokita, Krutika S. Gaonkar, Bailey Farrow, Nick VanKuren     |
| Supervision                |   Adam C. Resnick, Yuankun Zhu, Jo Lynne Rokita,  Angela Waanders,  Hongbo M. Xie, Cassie N. Kline  |
| Validation                 |   Mariarita Santi, Angela Viaene, Derek Hanson, Payal Jain |
| Visualization              |         |
| Writing – original draft   |    Jo Lynne Rokita, Krutika S. Gaonkar, Shrivats Kannan, Komal S. Rathi, Miguel A. Brown, Jessica Wong, Mateusz P. Koptyra     |
| Writing – review & editing |         |

Collecting ORCIDs [here](https://docs.google.com/spreadsheets/d/1tGU4BmeLuRQf2rl-f3NiVtT7h6Pp51aHoL63Jq1K3TI/edit#gid=0).

**Comment by jharenza** [comment]
cc @yuankunzhu, @allisonheath, and @baileyckelly to add people from your teams

**Comment by yuankunzhu** [comment]
@jharenza, I think we can add 
- Miguel and Jessica Wong to "Writing – original draft" due to their work at https://github.com/AlexsLemonade/OpenPBTA-manuscript/commits?author=migbro and https://github.com/AlexsLemonade/OpenPBTA-manuscript/commits?author=wongjessica93
- Dan Miller to either "Formal analysis" or "Software" depends on how we interpret those roles, as he contributed for the fusion modules at https://github.com/AlexsLemonade/OpenPBTA-analysis/commits?author=dmiller15
- me under "Data curation" for the V1 to V11 release

**Comment by jharenza** [comment]
@yuankunzhu, thanks, will add. Also, think about the data processing (I think "Formal Analysis") for releases - I know Brian worked on a lot of that - what about Bo? I will also add Miguel for CNV algorithms (and possibly David for Theta2 calculations?).

**Comment by jharenza** [comment]
also cc @mkoptyra for your roles

**Comment by aadamk** [comment]
Below are my contributions for authorship:

1) 'Supervision' for work related to the development and oversight of the analysis toolkit 
2) 'Methodology' for development of a new TMB protocol (@tkoganti can be added under this section for implementation of the 'Friends of Cancer Research' harmonization standards) 
3) 'Methodology' for development of a standardized RNAseq batch correction methodology (with @komalsrathi, though she is already listed under 'methodology'; soon to be merged to d3b-analysis-toolkit repo).

**Comment by komalsrathi** [comment]
@jaclyn-taroni am I correct in saying that we did not combine the `polya + stranded` samples for any OpenPBTA analyses? If so, @aadamk we don't need to use the batch correction methodology for the OpenPBTA manuscript. So, you can remove (3) from above.

**Comment by jaclyn-taroni** [comment]
@komalsrathi Yes that is correct, we do not use any combined poly-A and ribodeplete manuscript in this project currently.

**Comment by aadamk** [comment]
I see. 
As I understood, D3b was looking to merge modularized code for different DNA + RNA analyses. My thoughts were that this would include @komalsrathi's batch correction module to enable researchers to batch correct on the basis of site, as PBTA is inclusive of CBTTC + PNOC003 (and possibly 008 samples) in this release. 
Though do correct me if I am mistaken.

**Comment by jaclyn-taroni** [comment]
There is no pull request to include the batch correction module as of yet. 

> As I understood, D3b was looking to merge modularized code for different DNA + RNA analyses.

Can someone from D3b file an issue on https://github.com/AlexsLemonade/OpenPBTA-analysis so we have an idea of what to expect please? I think that will help us all get on the same page.

**Comment by komalsrathi** [comment]
@yuankunzhu please clarify if there are any plans for the above and if there has been any correspondence with CCDL about incorporating one or more modules from the D3b toolkit.

**Comment by mkoptyra** [comment]
Below are my contributions for authorship:
1. Formal Analysis - Molecularly subtype chordomas
2. Writing - Methods sections: Biospecimen collection and Nucleic acids extraction and library preparation

**Comment by yuankunzhu** [comment]
> @yuankunzhu please clarify if there are any plans for the above and if there has been any correspondence with CCDL about incorporating one or more modules from the D3b toolkit.

I think the plan is to use the current data release and existing analyses to prepare the manuscript, while for other modules and tools, we can bring them in after the manuscript drafting and keep iterating from there. That's my take away from our recent OpenPBTA call, but wanted to check if that's a shared understanding @jaclyn-taroni  @jharenza @aadamk 🙂.

**Comment by jharenza** [comment]
> > @yuankunzhu please clarify if there are any plans for the above and if there has been any correspondence with CCDL about incorporating one or more modules from the D3b toolkit.
> 
> I think the plan is to use the current data release and existing analyses to prepare the manuscript, while for other modules and tools, we can bring them in after the manuscript drafting and keep iterating from there. That's my take away from our recent OpenPBTA call, but wanted to check if that's a shared understanding @jaclyn-taroni @jharenza @aadamk 🙂.

I think that's right. @aadamk, I added you to Methodology for now, since that is definitely being used for V1 of the paper, and if we add the modules/batch correction, we can add those contributions later.

**Comment by aadamk** [comment]
Got it - makes sense @yuankunzhu and @jharenza. Thanks!

**Comment by jharenza** [comment]
Tracking authorship contributions in more detail here:

Jo Lynne Rokita (@jharenza)
- Conceptualization - conceiving open analysis via GitHub
- Data curation - [pbta-histologies.tsv V1-V15, data releases](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+is%3Aclosed+author%3Ajharenza) 
- Formal analysis - run initial GISTIC analysis
- Funding acquisition - ALSF Catalyst Award
- Investigation - ControlFreeC and CNVKit copy number algorithm benchmarking, ploidy analysis, QC histologies file/matrices
- Methodology - generate rules for molecular subtyping tickets, fusion analysis methods
- Software - annoFuse as basis for fusion filtering and prioritization
- Supervision - oversee D3b sprints to achieve upstream processing tasks and data releases, oversee fusion analysis/molecular subtyping
- Writing – original draft - [methods PRs](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pulls?q=is%3Apr+is%3Aclosed+author%3Ajharenza)

Krutika S. Gaonkar (@kgaonkar6)
- Data curation - create merge matrices for data releases
- Formal Analysis - [fusion analysis, TP53/NF1 classifier analysis, LGAT subtyping, EWS tumor reclassification](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+author%3Akgaonkar6+)
-  Investigation - QC and create QC code for matrix generation of RNA-Seq files
- Methodology - generate code and rules for fusion filtering and prioritization
- Software - created annoFuse as basis for fusion filtering and prioritization
- Writing – original draft - [fusion methods](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/21)

Pichai Raman (@PichaiRaman)
- Conceptualization - created many of the initial analysis request tickets
- Formal Analysis - [first iteration of ssGSEA](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/133)
- Methodology - create initial merge matrices for RNA-Seq data

Komal S. Rathi (@komalsrathi 
- Formal Analysis - [collapse RNA-Seq](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+is%3Aclosed+author%3Akomalsrathi+rnaseq) and [immune deconvolution](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+is%3Aclosed+author%3Akomalsrathi+immune+)
- Investigation - identify and fix bugs in matrix files (collapse RNA-Seq module)
- Methodology - collapse RNA-Seq
- Writing – original draft - [collapse RNA-Seq and immune deconvolution methods](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/64)

Shrivats Kannan (@shrivatsk)
- Formal Analysis - [sex determination from WGS](https://github.com/d3b-center/sex-determination-tool)
- Writing – original draft - [methods for sex determination from WGS
](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/40)

**Comment by jharenza** [comment]
Nhat Duong (@fingerfen)
- Formal Analysis - [Copy number consensus](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+is%3Aclosed+author%3Afingerfen+)
- Investigation - related to CNV consensus
- Methodology - generated method for CNV consensus

Hongbo M. Xie (@xiehongbo)
- Methodology - generated method for CNV consensus

 Mariarita Santi and Angela Viaene
- Investigation - [investigated pathology for samples that were reclassified genomically](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/609#issuecomment-602821376)
- Validation - determined proper diagnosis based on above investigation

Teja Koganti (@tkoganti) - please let me know your full name for publication
- Data curation - generation of merge matrices for data releases
- Formal Analysis - [TMB analysis, EPN subtyping](https://github.com/AlexsLemonade/OpenPBTA-analysis/pulls?q=is%3Apr+is%3Aclosed+author%3Atkoganti)
- Investigation - investigation of TMB

Adding additional contributions:
Miguel A. Brown (@migbro)
- Data curation - generation of merge matrices for data releases
- Formal analysis - MendQC, ControlFreeC, CNVKit, Vardict, Lancet, Strelka2, Mutect2
- Investigation - benchmarking of variant callers, CNV callers
- Methodology - generating best practice workflows for above
- Writing – original draft - [methods](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pulls?q=is%3Apr+is%3Aclosed+author%3Amigbro)

Brian M. Ennis (@bmennis) 
- Data curation - generation of merge matrices for data releases
- Formal analysis - initial LUMPY analysis, Manta SV (?) - please confirm

Bo Zhang (@zhangb1)
- Data curation - generation of merge matrices for data releases
- Formal analysis - STAR, STAR-Fusion, Arriba analyses

**Comment by baileyckelly** [comment]
Bailey Farrow & Nick VanKuren
- Data Curation
- Software

**Comment by jharenza** [comment]
Derek Hanson (D3b collaborator)
- Validation ([ETMR subtyping](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/251#issuecomment-571807158))

**Comment by jharenza** [comment]
@fingerfen, will you please add your email to [this google doc](https://docs.google.com/spreadsheets/d/1tGU4BmeLuRQf2rl-f3NiVtT7h6Pp51aHoL63Jq1K3TI/edit?usp=sharing) for authorship purposes? thanks!

**Comment by allisonheath** [comment]
Allison P. Heath
- Project administration

**Comment by jainpayal022** [comment]
Thanks @jharenza, on my end it will be the following:

1. Data curation - Compiled and collated PNOC003-related data along with supervising WGS data generation.
2. Validation - of fusion related output from annoFuse, LGG and HGG domain specific knowledge contribution
3. Investigation - QC of manifest building clinical and histologies file for V16/V17
4. Writing – review & editing of next manuscript iteration

**Comment by jharenza** [comment]
> Thanks @jharenza, on my end it will be the following:
> 
> 1. Data curation - Compiled and collated PNOC003-related data along with supervising WGS data generation.
> 2. Validation - of fusion related output from annoFuse, LGG and HGG domain specific knowledge contribution
> 3. Investigation - QC of manifest building clinical and histologies file for V16/V17
> 4. Writing – review & editing of next manuscript iteration

Thanks @jainpayal022 - I will leave off 4 right now, as I had noted those who had already written some pieces. We can add this contribution later.

**Comment by syzheng** [comment]
For us, 
Data analysis - telomerase activity prediction.
writing - contributing the corresponding method description.

**Comment by jaclyn-taroni** [comment]
@syzheng can you add your contributions, affiliations, and contact info to #115 and use the specific roles from CRediT when you do please? The available roles are (link to the definitions is included on #115):

```
Conceptualization
Data curation
Formal Analysis
Funding acquisition
Investigation
Methodology
Project administration
Resources
Software
Supervision
Validation
Visualization
Writing – original draft
Writing – review & editing
```

**Comment by syzheng** [comment]
@jaclyn-taroni @NNoureen 
Sure. 
Contribution:
Formal Analysis: we contributed telomerase activity analysis
Writing – Original Draft: we contributed description of telomerase prediction methods and made figures to show distribution of telomerase activity across disease entities. 

Affiliations: 
Greehey Children's Cancer Research Institute, UT Health San Antonio

Contact info:
Siyuan Zheng, zhengs3@uthscsa.edu

**Comment by NNoureen** [comment]
@jaclyn-taroni @syzheng

Affiliations: 
Greehey Children's Cancer Research Institute, UT Health San Antonio

Contact info:
Nighat Noureen
Initial: N.N.
Email:  noureen@uthscsa.edu
ORCID:  0000-0001-7495-8201

**Comment by jharenza** [comment]
Cassie N. Kline
- Supervision - informing `CNS_region` in [V17](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/732) `pbta-histologies.tsv
- Validation of histologies nomenclature

**Comment by jharenza** [comment]
Angela Waanders
- Supervision - informing [HGG subtyping](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/249), `glioma_brain_region`, and overall histology nomenclature

**Comment by jharenza** [comment]
closed with #111 and #122
